src/components/masters/program/main-content/course-cards/BaseCourseCard.jsx

```diff
@@ -82,6 +82,47 @@ class BaseCourseCard extends Component {
     return (
       <div className={classNames('card mb-4', { 'is-micromasters': !!microMastersTitle })}>
         <div className="card-body">
+          <div className="float-right mt-3 mt-xl-0">
+            <Dropdown>
+              <Dropdown.Button>
+                <FontAwesomeIcon icon={faCog} />
+              </Dropdown.Button>
+              <Dropdown.Menu>
+                <Dropdown.Item
+                  type="button"
+                  onClick={() => {
+                      this.setModalState({
+                        key: 'emailSettings',
+                        open: true,
+                        options: {
+                          title,
+                          hasEmailsEnabled,
+                        },
+                      });
+                      this.setState({
+                        hasNewEmailSettings: false,
+                      });
+                      sendTrackEvent('edx.learner_portal.email_settings_modal.opened', { course_run_id: courseRunId });
+                    }}
+                >
+                    Email Settings
+                  <span className="sr-only">for {title}</span>
+                </Dropdown.Item>
+                <Dropdown.Item
+                  type="button"
+                  onClick={() => {}}
+                >
+                    Move to completed
```

Comment:
i think we will probably not have move to completed or unenroll for the masters version of this dropdown, so it seems like these items should be passed in somehow so that each version of the page can control what dropdown items they want.

or, at the very least, these should be removed for now until we need to add them (and the backend functionality backing them) to the enterprise version of the page.

Reply:
I can definitely remove it for now. My understanding is that @adamstankiewicz will have a PR up that addreses the functionality and UI of the Basecoursecard. I only added the dropdown here to ensure that the functionality of EmailSettingsModal component worked properly in a dropdown component.

Reply:
@brittneyexline @erinakii Yeah, with the Layout refactor PR that provides page data through React's Context API to any children components, we'll be able to know whether a page is enterprise or not, and can add in the enterprise-specific dropdown items once that PR lands.